src/tutorials/0008-introduction-to-libp2p/06.md

```diff
@@ -0,0 +1,57 @@
+---
+    title: "Applications and Services using libp2p"
+    type: "text"
+---
+
+There are a lot of applications and services using `libp2p` at the moment. Here we list some examples, but the list continues to grow as you read this.
```

Comment:
Could we add some more detail here (or after the logos) to capture the breadth of use cases that libp2p has been applied to? Giving some examples of what these applications do could help get across the point you made earlier about the importance of not trapping a networking protocol inside a specific use case.

Reply:
Would need some help with this + I think we need to review the list or consider if we should have this list here at all: some might drop support in the future.

@yusefnapora what do you think?

this is my initial classification, by I don't think I got them correctly and I didn't know where to put others:

- protocol: ipfs, matrix
- file storage: filecoin, peergos
- video streaming: livepeer
- crypto wallet / identity: metamask
- dev tools: textile, ethereum, 0x
- blockchain: ethereum, polkadot
- marketplace: openbazaar